.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
```

Comment:
Yep, I've had this discussion before and in many years of development, I've never seen a node_modules committed to any node repo I maintain. And amongst the challenges for newcomers, this is extremely minor.